versions.json

```diff
@@ -0,0 +1,25 @@
+{
+    "CoreData Desktop": {
+	"version": "3.0.1",
+	"important": true,
+	"minVersion": "2.5.1",
+	"binary": "http://coredata.is/setup/coredata_desktop-${ARCH}-3.0.1.0.msi"
+    },
+
+    "CoreData Desktop .NET40": {
+        "version": "3.0.1",
+        "important": true,
```

Comment:
CoreData Desktop updates can be marked as important, if they are then a notification will be shown to customers running CoreData Desktop versions that are older. Without 'important' flag the customer would have to run 'check for updates' to see the update.